Automattic/jetpack
# Slideshow: Replace pencil icon with edit text in toolbar

Closes #27711 

#### Changes proposed in this Pull Request:

* Replace `EditButton` with `Button`.
* Use `Edit` as `Button` text.

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Create a JN site
* Connect Jetpack
* Create a new post with a Slideshow
* Insert images.
* Click on the Slideshow block.
* It should have the `Edit` option and not a `pencil`.

### Demo 💻 

![image](https://user-images.githubusercontent.com/1663717/208469114-58927aa2-5e88-4216-85a0-8f7e127eea2b.png)
